remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -81,9 +89,22 @@
     public void setup()
     {
         given(authorityServiceMock.hasAdminAuthority()).willReturn(true);
+        given(nodesMock.validateNode(NODE_ID)).willReturn(TAG_NODE_REF);
         given(nodesMock.validateNode(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, TAG_ID)).willReturn(TAG_NODE_REF);
+        given(nodesMock.getNode(any())).willReturn(createdNodeMock);
+        given(createdNodeMock.getNodeId()).willReturn(NODE_ID);
+        given(typeConstraint.matches(any())).willReturn(true);
         given(taggingServiceMock.getTagName(TAG_NODE_REF)).willReturn(TAG_NAME);
+        given(taggingServiceMock.findTaggedNodesAndCountByTagName(any())).willReturn(createTagsCountPairList());
     }
+
+    public List<Pair<String,Integer>> createTagsCountPairList()
```

Comment:
I guess this doesn't need to be public?
```suggestion
    private List<Pair<String,Integer>> createTagsCountPairList()
```

Reply:
Removed the method